cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMaker2AssemblyOperator.java

```diff
@@ -367,6 +368,9 @@ private static void addClusterToMirrorMaker2ConnectorConfig(Map<String, Object>
             if (KafkaClientAuthenticationPlain.TYPE_PLAIN.equals(clientAuthType)) {
                 saslMechanism = "PLAIN";
                 jaasConfig = "org.apache.kafka.common.security.plain.PlainLoginModule required username=\"" + authProperties.get(AuthenticationUtils.SASL_USERNAME) + "\" password=\"${file:" + CONNECTORS_CONFIG_FILE + ":" + cluster.getAlias() + ".sasl.password}\";";        
+            } else if (KafkaClientAuthenticationScramSha256.TYPE_SCRAM_SHA_256.equals(clientAuthType)) {
+                saslMechanism = "SCRAM-SHA-256";
+                jaasConfig = "org.apache.kafka.common.security.scram.ScramLoginModule required username=\"" + authProperties.get(AuthenticationUtils.SASL_USERNAME) + "\" password=\"${file:" + CONNECTORS_CONFIG_FILE + ":" + cluster.getAlias() + ".sasl.password}\";";
```

Comment:
I'm a bit puzzled why is this change needed only in the MirrorMaker2 related class. Why is it not needed for Connect or Bridge or Mirror Maker 1? Are you sure this is the only place where this is needed? (not saying it is, I'm really just wondering)

Reply:
The other cases are handled by polymorphism (inheriting from the common superclass ```io.strimzi.api.kafka.model.authentication.KafkaClientAuthenticationScram```)